client/js/hah-app/reducers.js

```diff
@@ -27,6 +27,7 @@ const authenticationReducer = (state = {}, action) => {
 const resourceReducer = (state = {}, action) => {
 	switch (action.type) {
 		case GET_RESOURCE_TO_VERIFY:
+			state.hotlinePhoneNumber = action.resource.hotlinePhoneNumber
```

Comment:
@bhavyaab We can't update state directly in redux 
https://redux.js.org/recipes/structuringreducers/immutableupdatepatterns

Don't worry about this not updating correctly. I'm working on a separate PR to clear the entire form on save and when you load a new resource. We keep running into this problem and it's a really major issue so I want to make sure it's solved for every single field on the form.